### PR TITLE
test: select category have to keep it, on category url entry.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#303

### CODE REVIEW

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -69,15 +71,15 @@ public function iShouldWaitForTheTextTo($text, $appear) {
    */
   public function iWaitForCssElement($element, $appear) {
     $xpath = $this->getSession()->getSelectorsHandler()->selectorToXpath('css', $element);
+    print_r($xpath);
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove.

File: behat/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -115,9 +117,16 @@ public function deleteRestfulTokens($event) {
       // No tokens found.
       return;
     }
+    // Delete from database.
     foreach ($entities as $entity) {
       $entity->delete();
     }
+    // Reload after visit root path, to force the logout.
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: remove

File: behat/features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -137,11 +146,11 @@ public function resizeWindow() {
    * @param $fn
    *   A callable to invoke.
    * @param int $timeout
-   *   The timeout period. Defaults to 60 seconds.
+   *   The timeout period. Defaults to 30 seconds.
    *
    * @throws Exception
    */
-  private function waitFor($fn, $timeout = 60000) {
+  private function waitFor($fn, $timeout = 30000) {
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: 5000

File: negawatt/modules/custom/negawatt_restful/plugins/formatter/electricity_hal_json/NegawattFormatterElectricityHalJson.class.php [hunk 4]
```diff
@@ -22,18 +22,21 @@ public function prepare(array $data) {
...
-      }
-
-      if ($meter_id !== $row['meter']->nid) {
-        $meter_id = $row['meter']->nid;
-      }
-
-      $output['total'][$month]['kwh'] += $row['kwh'];
-      $month++;
-    }
+//    foreach ($data as $row) {
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/formatter/electricity_hal_json/NegawattFormatterElectricityHalJson.class.php]: remove

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**